### ISSUE TITLE
signup page - post T&C sign, page doesn't auto login

### ISSUE THREAD

**Issue by nc-2k** [twali-xyz/drippi#94]
it closes the box, then sits on the signup page. Should log the user in automatically 

I had to go to 'login', enter my details and click login to access dashboard

**Comment by hone1er** [comment]
Pushing user to Login page for now until can implement full flow. With 2FA we need a verification code to login the user now, even just after signup @nc-2k

**Comment by ivyastrix** [comment]
requiring signin after login is a security best practice - assuming the bug of going to the signup page rather than signin is fixed tho @hone1er ?

**Comment by hone1er** [comment]
@ivyastrix yup it is working as you described. Goes to the signin page after signup
